tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraLongLivedTokenSetMethodContract.scala

```diff
@@ -371,4 +524,75 @@ trait LinagoraLongLivedTokenSetMethodContract {
          |}""".stripMargin)
   }
 
+  @Test
+  def methodShouldSupportBackReferences(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "create": {
+         |        "K38": {
+         |          "deviceId": "My android device"
+         |        }
+         |      }
+         |    }, "c1"],
+         |   ["Core/echo", {
+         |      "arg1": "#K38"
+         |    }, "c2"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |    "sessionState": "${SESSION_STATE.value}",
+         |    "methodResponses": [
+         |        [
+         |            "LongLivedToken/set",
+         |            {
+         |                "accountId": "$ACCOUNT_ID",
+         |                "created": {
+         |                    "K38": {
+         |                        "id": "$${json-unit.ignore}",
+         |                        "token": "$${json-unit.ignore}"
+         |                    }
+         |                }
+         |            },
+         |            "c1"
+         |        ],
+         |        [
+         |            "Core/echo",
+         |            {
+         |                "arg1": "$${json-unit.ignore}"
```

Comment:
We could assert that arg1 value is not `#K38` ?

Reply:
I think with this code (it already), it is ok
```scala
    val longLivedTokenId:String = (((Json.parse(response) \\ "methodResponses" )
      .head \\ "created")
      .head \\ "id")
      .head.asInstanceOf[JsString].value

    val backReferenceValue:String = ((Json.parse(response) \\ "methodResponses" )
      .head \\ "arg1")
      .head.asInstanceOf[JsString].value

    assertThat(longLivedTokenId)
      .isEqualTo(backReferenceValue)
```

Reply:
Yes it is.